core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1607,6 +1708,22 @@ class TopicChangeListener(controller: KafkaController, eventManager: ControllerE
   }
 }
 
+/**
+  * Called when broker notifies controller of disk change
+  *
+  * @param controller
+  */
+class LogDirEventNotificationListener(controller: KafkaController, eventManager: ControllerEventManager) extends IZkChildListener with Logging {
```

Comment:
@ijuma Thanks. I think I will keep the existing approach of including `ControllerEventManager` in the constructor of those listeners since it is needed in the test. We can refactor them in a separate patch if it is necesssary.